examples/discovery-search-app/cypress/support/commands.ts

```diff
@@ -1,3 +1,5 @@
+export {}; //forces this file to be considered a module, which prevents a TS compilation error
```

Comment:
nitpick - would this be better at the bottom?

Reply:
Doesn't matter to me. If #45 gets merged before this PR, then I don't think we'll even need this line.